frontend/packages/kubevirt-plugin/integration-tests/v2v/rhv/v2v.rhv.configs.ts

```diff
@@ -41,6 +41,7 @@ export const rhvVMConfig: VMImportConfig = {
   sourceVMName: vmName,
   provider: RHV,
   instanceConfig: newInstanceConfig,
+  startOnCreation: true,
```

Comment:
We should also wait for `Running` status in v2v.rhv.import.scenario.ts where we test this config.

Maybe it would be better to have two tests - one for `Off` and one for `Running`

Reply:
Do you mean running VM? I don't expect VM to run during migration in this test

Reply:
I mean on side of RHV

Reply:
no I mean on kubevirt side:

we could have two test cases 

`startOnCreation: true,`

and 

`startOnCreation: false,`

Reply:
This parameter is optional, each test where this parameter is not specified will work as your second line

Reply:
Added separate tests, like you requested

Reply:
thanks!